### PR TITLE
cli: add new translate command

### PR DESCRIPTION
**Pull request by rerowep**
* Adds cli command translate for automatic translation of languages.
* Adds tranlation of englisch languages from LOC.
* Adds creation of title map for languages.

Co-Authored-by: Peter Weber <peter.weber@rero.ch>
Co-Authored-by: Olivier DOSSMANN <git@dossmann.net>

## Why are you opening this PR?

- task https://tree.taiga.io/project/rero21-reroils/task/1167

## How to test?

- test language translations in Documents creation

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/cli.py [hunk 1]
```diff
@@ -930,3 +935,182 @@ def reindex(pid_type):
...
+                click.echo('\t{')
+                click.echo('\t\t"value": "{value}",'.format(value=value))
+                click.echo('\t\t"name": "{value}"'.format(value=value))
+                if value == values[-1]:
+                    click.echo('\t}')
+                else:
+                    click.echo('\t},')
+            click.echo(']')
+
+    def chnage_po(po, values):
```
Comment (jma) [inline rero_ils/modules/cli.py]: spell

**Comment by AoNoOokami** [comment]
Typo in the commit message:
* Adds tranlation of englisch languages from LOC. -> translation, english

File: rero_ils/modules/cli.py [hunk 2]
```diff
@@ -930,3 +935,176 @@ def reindex(pid_type):
...
+@utils.command('translate')
+@click.argument('translate_to', type=str)
+@click.option('-c', '--change', 'change', is_flag=True, default=False)
+@click.option('-t', '--title_map', 'title_map', is_flag=True, default=False)
+@click.option('-l', '--no_loc_en', 'no_loc_en', is_flag=True, default=True)
+@click.option('-v', '--verbose', 'verbose', is_flag=True, default=False)
+def translate(translate_to, change, title_map, no_loc_en, verbose):
+    """Automatic language, country, canton, language scrip translations."""
+    def print_title_map(name, values):
+        """Print out a titel map."""
```
Comment (AoNoOokami) [inline rero_ils/modules/cli.py]: Typo: title map

File: rero_ils/modules/cli.py [hunk 3]
```diff
@@ -930,3 +935,176 @@ def reindex(pid_type):
...
+                )
+                entry.msgstr = values[entry.msgid]
+                if entry.fuzzy:
+                    entry.flags.remove('fuzzy')
+
+
+    try:
+        locale = Locale(translate_to)
+
+        click.secho('Translate countrycodes to: {translate_to}'.format(
```
Comment (AoNoOokami) [inline rero_ils/modules/cli.py]: country codes?

**Pull request merged**